### DISCUSSION TITLE
Superkey vs package name/UID check

### DISCUSSION THREAD

**Discussion by axelkar** [bmax121/APatch#644] [General]
Why is there a "superkey"? Why not a Magisk manager-like thing where APatch checks the package name/UID?

Wouldn't it be way more secure to let the user just toggle apps access to root? Maybe the UIDs could be fetched by the manager and saved to a special file, which the kernel module would read.

**Comment by axelkar** [comment]
@bmax121

**Comment by bmax121** [comment]
The KernelPatch was originally designed to not require APatch, and it still does

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
